test/nntrainer_test_util.cpp

```diff
@@ -242,8 +242,7 @@ nntrainer::GraphRepresentation makeCompiledGraph(
     model_graph.addLayer(layer);
   }
 
-  // Compile with loss
-  model_graph.compile("mse");
+  model_graph.compile("");
```

Comment:
Could you please describe the reason of this removal in the commit message?

If you just wanted to test the case without ```mse```, can you add another case without mse and keep this case? (If you've removed mse because it's "incorrect" to have mse here, it's ok.)